src/components/DiffView/index.spec.tsx

```diff
@@ -43,4 +45,25 @@ describe(__filename, () => {
       expect(root.find(Diff).at(index)).toHaveProp('hunks', diff.hunks);
     });
   });
+
+  it('renders a header with diff stats', () => {
+    const root = render();
+
+    expect(root.find(`.${styles.header}`)).toHaveLength(1);
+    expect(root.find(`.${styles.stats}`)).toHaveLength(1);
+    expect(root.find(`.${styles.stats}`)).toIncludeText('+++ 2--- 0');
+  });
+
+  it('renders hunks with separators', () => {
+    const parsedDiff = parseDiff(diffWithDeletions)[0];
+
+    const root = shallow(<DiffView diff={diffWithDeletions} />);
+    const diff = root.find(Diff).shallow();
```

Comment:
This works for me, although if the lib changes the interface, we won't notice it, right? I am thinking about `children` being a function. It's weird and maybe it will evolve?

Reply:
Expecting `children` to be a function is a common [render prop technique](https://reactjs.org/docs/render-props.html#using-props-other-than-render) so it doesn't seem weird to me and I'm not too worried about it changing.